webserver/views/api/v1/dataset_eval.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import absolute_import
+from flask import Blueprint, jsonify, request
+from flask_login import current_user
+from webserver.decorators import auth_required
+import db.dataset_eval
+
+bp_dataset_eval = Blueprint('api_v1_dataset_eval', __name__)
+
+@bp_dataset_eval.route("/pending-jobs-for-user", methods=["GET"])
```

Comment:
Don't forget to document all the endpoint properly. See existing ones for an example. In this case also mention that only pending jobs for authenticated user are returned.